src/components/pages/NewRisk/NewRisk.js

```diff
@@ -62,81 +62,99 @@ function NewRisk(props) {
   const [riskScore, setRiskScore] = useState(2);
   const [riskLocation, setRiskLocation] = useState(mapCentre);
```

Comment:
Pass object in instead of mapCentre:
`{
  lat: 52.475,
  lng: -1.9,
}`

Reply:
object passed through instead of variable